.github/workflows/ci.yml

```diff
@@ -53,9 +62,19 @@ jobs:
           key: ${{ runner.os }}-gradle-home-${{matrix.gradle_args}}_check-${{ hashFiles('**/*.gradle') }}
       - name: Clear existing docker image cache
         run: docker image prune -af
+      - name: Thundra Gradle Test Instrumentation
+        if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' }}
+        uses: thundra-io/thundra-gradle-test-action@v1
+        with:
+          apikey: ${{ secrets.THUNDRA_API_KEY }}
+          project_id: ${{ secrets.THUNDRA_PROJECT_ID }}
       - name: Build and test with Gradle (${{matrix.gradle_args}})
         run: |
-          ./gradlew --no-daemon --continue --scan ${{matrix.gradle_args}}
+          ./gradlew --no-daemon --continue --scan ${{matrix.gradle_args}} \
+            $($THUNDRA_GRADLE_INIT_SCRIPT_PATH || echo "--init-script $THUNDRA_GRADLE_INIT_SCRIPT_PATH")
```

Comment:
This is the easiest way to understand if the Thundra step was successful and if so, instrument the build. If not, we don't want to break the build. Hence the check. 

The template of the init script can be found here: https://github.com/thundra-io/thundra-gradle-test-action/blob/master/templates/thundra.gradle.ejs